app/src/main/java/com/gathering/android/auth/password/forgetPassword/ForgetPasswordViewModel.kt

```diff
@@ -51,36 +55,30 @@ class ForgetPasswordViewModel @Inject constructor(
     }
 
     fun onSendLinkBtnClicked(email: String) {
+        viewModelState.update { currentViewState ->
```

Comment:
good job taking this out of coroutine. this probably creates some timespan between the first and second emissions and that stops emission loss by stateflow.